resources/function-controller/templates/cluster-microfrontend-functions.yaml

```diff
@@ -1,28 +1,32 @@
-{{ if .Values.clusterMicroFrontend.enable }}
+{{- if .Values.clusterMicroFrontend.enabled }}
 apiVersion: "ui.kyma-project.io/v1alpha1"
 kind: ClusterMicroFrontend
 metadata:
   name: serverless
   labels:
-    app:  serverless
     {{- include "tplValue" ( dict "value" .Values.global.commonLabels "context" . ) | nindent 4 }}
 spec:
   displayName: Functions
   version: v1
-  category: Experimental
+  category: Development
   placement: namespace
   viewBaseUrl: 'https://core-ui.{{ js .Values.global.ingress.domainName }}'
   navigationNodes:
-    - label: Functions [preview]
-      navigationPath: functions-preview
+    - label: Functions
+      navigationPath: functions
       viewUrl: '/lambdas'
       order: 1
-    - label: Lambda Details
-      navigationPath: functions-preview/details
+      viewGroup: "_core_ui_"
+      requiredBackendModules:
+        - serverless
```

Comment:
is it the only one?

Reply:
Yes, another like `service-catalog` etc is optional, because we handle that in code :)

Reply:
Yes, another like servicecatalog, application etc are optional. We handle existing them on UI in code :)